recipe/meta.yaml

```diff
@@ -32,6 +32,7 @@ requirements:
     - typing_extensions >=3.10.0.0
 #    - windows-curses  # [win]
     - wrapt
+    - msgpack-python
```

Comment:
If the dependency is not added the pip check fails as shown in https://github.com/conda-forge/python-can-feedstock/pull/17

Reply:
If `msgpack-python` is optional but ill defined in the package metadata as such, it is fine to skip pip-check. However, downstream packages should also drop it. Ideally we should fix the pkg metadata and declare it optional. However, if the upstream devs want to make it a hard requirement the solution here is good to merge.